nx-X11/programs/Xserver/hw/nxagent/man/nxagent.1

```diff
@@ -161,6 +161,11 @@ sets key-click volume (allowable range: 0-100).
 sets the visual class for the root window of color screens.
 The class numbers are as specified in the X protocol.
 Not obeyed by all servers.
+.TP 8
+.B \-co \fIfilename\fP
+This used to be the option for specifying the name of RGB color
+database. As the RGB color database is now integrated this option has
+no effect but is kept for compatibility.
```

Comment:
there should be a big DEPRECATED: here... Maybe under a separate headline...

Reply:
We have the bc option above which has  "Deprecated." I have done the same now.